examples/passive_scalar/benchmark/cylinder2d/args.template

```diff
@@ -0,0 +1,6 @@
+nelem = 40
+length_usr = 128
+level_usr = math.log(length_usr)/math.log(2)
+t_total = 10
+interval = t_total / 10
+stl_path = '$!stl_path!$'
```

Comment:
please provide default stl_path and use recheck (shepherd) specific parameter block `$!stl_path!$` only when running this testcase using shepherd. 
For example:
```
shepherd = true
if shepherd then
  stl_path = '$!stl_path!$'
else
  stl_path = './cylinder40.stl'
end
```

See example `examples/fluid/absorbingLayer/acousticCylinder2D/seeder.lua`

Reply:
No need for that here. The `args.template` is only used by shepherd anyway. If run manually we are including `args.lua` instead.